ra_aid/tools/shell.py

```
@@ -68,11 +75,13 @@ def run_shell_command(command: str) -> Dict[str, Union[str, int, bool]]:
         print()
         output, return_code = run_interactive_command(["/bin/bash", "-c", command])
         print()
-        return {
+        result = {
             "output": truncate_output(output.decode()) if output else "",
             "return_code": return_code,
             "success": return_code == 0,
         }
+        log_work_event(f"Executed shell command: {_truncate_for_log(command)}")
+        return result
```

Ruff (0.8.2)

Consider moving this statement to an `else` block

(TRY300)